src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -98,6 +98,12 @@ default_maxvcpus = @DEFMAXVCPUS_DB@
 # > 5                --> will be set to 5
 default_bridges = @DEFBRIDGES@
 
+# Enable balloon f_reporting, default false
+# Enabling this will result in the VM balloon device having f_reporting=on set
+#
+# Default false
+#enable_balloon_f_reporting = true
+
```

Comment:
Btw, is this feature exclusive to the Dragonball? If not, please update other configs.

Reply:
> Btw, is this feature exclusive to the Dragonball? If not, please update other configs.

It just support dragonball now.